blocks/hero-carousel/hero-carousel.css

```diff
@@ -120,7 +125,7 @@
 }
 
 .hero-carousel .carousel-slide-content .carousel-slide-description em {
-  color: var(--main-cta-background);
```

Comment:
Can you change the main-cta-background variable in the styles.css file because I was using it in other blocks as well instead of using the a new variable?

Reply:
@mind-adobe Done, please check now.